src/ngResource/resource.js

```diff
@@ -559,6 +559,10 @@ angular.module('ngResource', ['ng']).
           return ids;
         }
 
+        function defaultRequestInterceptor(httpConfig) {
```

Comment:
With the change proposed in https://github.com/angular/angular.js/pull/13273/files#r53487712, this is obsolete.